tasks/setup.yml

```diff
@@ -4,8 +4,6 @@
   file:
     path: "{{ magento_release_folder }}"
     state: "directory"
-    recurse: "yes"
```

Comment:
wonder why did we have the recurse to begin with and why when it started creating problems with symlinks it was fixed by adding no follow... could it be that there are some nasty problems on some projects and this is used as a dirty fix?

Reply:
actual reason why it was originally done like it was done is unknown to me, but maybe @toxix  can shed some light on the topic.

Also the suggested change was working in my project (don't bring names here) before I switch it to use this share role again.